acsm_compiler_flags.m4

```diff
@@ -585,6 +585,24 @@ AC_DEFUN([ACSM_SET_CXX_FLAGS],
                        ],
 
             [clang], [
+                       dnl On Darwin with clang + gfortran, we get very many warnings for compact unwinding issues
+                       dnl We deliberately keep relying on the less performant dwarf unwinding until the over-production of warnings is solved.
+                       OLD_ACSM_LDFLAGS="$ACSM_LDFLAGS"
+                       ACSM_LDFLAGS+=" -Wl,-femit-dwarf-unwind=no_compact_unwind"
```

Comment:
As I understand it, `AC_LINK_IFELSE` specifically uses the variables `LIBS` and `LDFLAGS` to test linking:

https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Running-the-Linker.html

So I think you need to do like in the xdr example, and set the `LIBS` variable prior to calling `AC_LINK_IFELSE`. Also, it seems to me that there is an issue with your test program, it will need to `#include <iostream>` in order to be well-formed.

Reply:
ok I changed it

Reply:
"-femit-dwarf-unwind=always" does not do much here btw, it s the two others that make the warnings disappear